test/sagemaker_tests/pytorch/training/integration/sagemaker/test_distributed_operations.py

```diff
@@ -61,49 +63,59 @@ def can_run_smmodelparallel_efa(ecr_image):
 @pytest.mark.skip_test_in_region
 def test_dist_operations_cpu(sagemaker_session, framework_version, ecr_image, instance_type, dist_cpu_backend):
     instance_type = instance_type or 'ml.c4.xlarge'
-    _test_dist_operations(sagemaker_session, framework_version, ecr_image, instance_type, dist_cpu_backend)
+    _test_dist_operations(ecr_image, sagemaker_session, framework_version, instance_type, dist_cpu_backend)
```

Comment:
Why is this test not run in multiple regions?

Reply:
The intension was to add multi region support for GPU only as we see capacity issues for p type instances. I will add the support for CPU as well